linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/correct_stig.pass.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
I have face issue as test scenario `correct_stig.pass.sh` for `accounts_password_set_max_life_existing`.